spritzle/resume_data.py

```diff
@@ -112,6 +112,11 @@ def save_torrent(self, torrent_handle):
                 self.save_torrent(torrent)
         await asyncio.gather(*self.resume_data_futures.values())
 
+    async def delete(self, torrent_name):
```

Comment:
Why is this a coroutine?

Reply:
It doesn't need to be. Guess just because everything else here was.

Reply:
Fixed.